### PR TITLE
Changes video thumbnail menu icon

### PR DESCRIPTION
**Pull request by yanas**

### CODE REVIEW

File: css/_vertical_filmstrip_overrides.scss [hunk 1]
```diff
@@ -65,7 +65,7 @@
             left: 0;
             top: auto;
             right: auto;
-            transform: translate3d(0,0,0) rotate(-90deg);
```
Comment (virtuacoplenny) [inline css/_vertical_filmstrip_overrides.scss]: Why remove the hardware acceleration?

Comment (yanas) [inline css/_vertical_filmstrip_overrides.scss]: Oups I just wanted to remove the rotation! I'll fix that

**Comment by virtuacoplenny** [comment]
Do you want the dots to be horizontally aligned in horizontal filmstrip mode?

**Comment by yanas** [comment]
No, I was considering it and decided against it for now :)

**Pull request merged**